torch/testing/_internal/common_jit.py

```diff
@@ -143,18 +143,78 @@ def getExportImportCopy(self, m, also_test_file=True, map_location=None):
             torch.jit.save(imported, fname)
             return torch.jit.load(fname, map_location=map_location)
 
+    def autoDiffErrorMessage(self, should_autodiff_node, nodes_not_in_diff_graph, 
+                             fusion_nodes_not_found, non_fusible_nodes_being_fused, 
+                             fusion_nodes_found, nodes_in_diff_graph):
+        err_msg = "\nFailure in testing nodes' autodifferentiation, "
+        if should_autodiff_node:
+            err_msg += "one or more nodes were expected to be autodiffed, " \
+                "but were not found in specified fusible/nonfusible " \
+                "DifferentiableGraph groups. \nSpecifically:"
+            for node in nodes_not_in_diff_graph:
+                err_msg += f"\n  {node} was not in one of the DifferentiableGraphs " \
+                    "when it was expected to be. Did you intend for this node to be " \
+                    "autodiffed? If not, remove it from the list of nonfusible nodes."
+                if node in non_fusible_nodes_being_fused:
+                    err_msg += "Additionally, This node was found in a FusionGroup " \
+                        "in a DifferentiableGraph. If that was intended, " \
+                        "reclassify this node as a fusible node. If not, your " \
+                        "autodifferention logic might be wrong."
+            for node in fusion_nodes_not_found:
+                err_msg += f"\n  {node} was not in one of the DifferentiableGraphs' " \
+                    "fusion groups when it was expected to be. " \
+                    "Did you intend for this node to be fused? If not, you should " \
+                    "move this node into the test's non-fusible nodes."
+        else: 
+            err_msg += "one or more nodes were not expected to be autodiffed, " \
+                "but were found in a fused/nonfused DifferentiableGraph group. " \
+                "Did you intend for these nodes to be autodiffed? " \
+                "If so, change this test to expect autodifferention. " \
+                "\nSpecifically:"
+            for node in fusion_nodes_found:
+                err_msg += f"\n  {node} was not expected to in one of the " \
```

Comment:
"to in one of the" -> "to be in one of the"

Reply:
Generally these errors mention multiple differentiable graphs, but this statement is written as if there's only one.